source/crud/examples/php/usage_example.php

```diff
@@ -234,7 +234,7 @@
             // Required update param
             ['$inc' => ['x' => 2]],
             // Optional named params in an associative array
-            ['upsert' => false],
+            ['upsert' => true],
```

Comment:
It occurred to me that these example files are outdated and should probably be deleted: https://jira.mongodb.org/browse/SPEC-1444

In any event, I think you can omit this change from the PR.

Reply:
Reverted this change.